N1990.Episerver.Cms.Audit/Infrastructure/ServiceCollectionExtension.cs

```diff
@@ -0,0 +1,14 @@
+﻿using N1990.Episerver.Cms.Audit.Business;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class ServiceCollectionExtensions
+    {
+        public static IServiceCollection AddAuditServices(this IServiceCollection services)
+        {
+            services.AddTransient <ICmsAuditor, CmsAuditor> ();
+
```

Comment:
Do we actually need to register it? We had that in the previous commit, but since this module no longer has any files in _protected folder (views are compiled into dll) we removed it.
![image](https://user-images.githubusercontent.com/106393/194857929-1acc2a6c-9012-481c-81bf-5a00ddcad537.png)